app/views/content_items/statistics_announcement.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, "#{@content_item.title}" %>
```

Comment:
Should the format be in the title?

Current titles look like:
`<title>Provisional Accident and Emergency Quality Indicators for England - October 2015, by provider   - Statistics release announcement - GOV.UK</title>`